### PR TITLE
Initial travis configuration

### PR DESCRIPTION
**Pull request by nelsonsar**
This is an initial TravisCI configuration. Most tests are failling but, we can start by fixing it and adding more as we iterate.

I changed Ruby version in Gemfile because Rails 3.0.x does not support Ruby 2.2.+.

I'll change the README file to indicate how to run tests but if you need to run we can do it running: `rake` in the root folder of the project :)

### CODE REVIEW

**Comment by nelsonsar** [comment]
Related to #172

File: README.md [hunk 1]
```diff
@@ -22,10 +22,16 @@ $> vimgolf put [challenge ID]
 
 # VimGolf.com web app
 
+[![Build Status](https://travis-ci.org/nelsonsar/vimgolf.svg?branch=web)](https://travis-ci.org/nelsonsar/vimgolf)
```
Comment (igrigorik) [inline README.md]: We'll need to update these to point to /igrigorik/vimgolf once merged?

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 source 'http://rubygems.org'
-ruby '2.3.1'
+ruby '2.1.5'
```
Comment (zoso10) [inline Gemfile]: any reason you bumped the ruby version down? it might be safer to make version changes separately.

Comment (nelsonsar) [inline Gemfile]: Yep. Without it rspec won't run since Rails 3.0.x does not support Ruby 2.2+ :(

**Comment by nelsonsar** [comment]
@igrigorik can you setup the repo in TravisCI so when its get merged the build runs? Another important point is: I'm not pretty sure the project will run fine with Ruby 2.1.5 (as pointed by @zoso10) is there an easy way to test if all funcionalities are working?

PS: I'm already working on model tests. Should I make different PRs for different models or can I put all in the same PR?

**Comment by igrigorik** [comment]
Enabled Travis. 

Re, Ruby version: I'll merge this, it's fine if it breaks the build.. I'm not going to push it live. Let's get this landed and start iterating on upgrading Rails.

**Comment by igrigorik** [comment]
p.s. 👍 thanks for taking this on!

**Comment by igrigorik** [comment]
Build: https://travis-ci.org/igrigorik/vimgolf/builds/187620397.. Looks like we have some work to do on the specs ;-)

File: Gemfile (hunk 2 above)
Comment (zoso10) [inline Gemfile]: hmm, I was able to run rails server locally with ruby 2.3.1 and rails 3.0.19. were you having problems?

Comment (nelsonsar) [inline Gemfile]:
```
cretan:vimgolf nelsonsar$ rake
/Users/nelsonsar/.rvm/gems/ruby-2.3.1/gems/activesupport-3.0.19/lib/active_support/values/time_zone.rb:272: warning: circular argument reference - now
/Users/nelsonsar/.rvm/rubies/ruby-2.3.1/bin/ruby -I/Users/nelsonsar/.rvm/gems/ruby-2.3.1/bundler/gems/rspec-core-98429a80a718/lib:/Users/nelsonsar/.rvm/gems/ruby-2.3.1/bundler/gems/rspec-support-436adb4b6d9b/lib /Users/nelsonsar/.rvm/gems/ruby-2.3.1/bundler/gems/rspec-core-98429a80a718/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb
/Users/nelsonsar/.rvm/gems/ruby-2.3.1/gems/activesupport-3.0.19/lib/active_support/values/time_zone.rb:272: warning: circular argument reference - now
Ruby 2.2+ is not supported on Rails 3.0.19. Check the Rails release notes for the appropriate update with support.
```

Comment (zoso10) [inline Gemfile]: TIL, thanks!

**Pull request merged**